seeq/addons/plot_curve/ui_components/templates/AppLayout.vue

```diff
@@ -51,7 +53,7 @@
                   <tbody>
                   <tr>
                     <td color='red'><b>Units</b></td>
-                    <td>m3/hr</td>
+                    <td>m^3/hr</td>
```

Comment:
Does it render correctly if you use superscript 3? (\u00b3). Might be more appealing on the frontend.

Reply:
I see. You're demonstrating the correct units entry format.  It makes sense to use the caret in that case.

Reply:
It should, but since this is the instructions, I'm intentionally trying to show the ^ symbol here in the UI, since in the csv file that will need to be present.

Reply:
haha yes - you beat me to the punch.